ako-gateway-api/nodes/avi_model_l7_translator.go

```diff
@@ -460,3 +467,39 @@ func (o *AviObjectGraph) BuildHTTPPolicySetHTTPRequestRedirectRules(key, httpPSn
 	}
 	return isRedirectPresent
 }
+
+func (o *AviObjectGraph) BuildHTTPPolicySetHTTPRequestUrlRewriteRules(key, httpPSname string, vsNode *nodes.AviEvhVsNode, routeModel RouteModel, filters []*Filter, index int) bool {
+	urlRewriteAction := &models.HTTPRewriteURLAction{}
+	isUrlRewritePresent := false
```

Comment:
Nit: We can avoid this variable by returning true or false.

Reply:
Removed the variable altogether